<launch>
  
  <!-- load controller configurations from YAML file to parameter server -->
  <rosparam file="$(find grips_control)/config/joint_position_controllers.yaml" command="load" />
  
  <!-- load the controllers -->
  <node name="controller_manager" pkg="controller_manager" type="spawner" output="screen" 
      args="SA SE linkage_tr WP WY WR"/>
  
  <node name="state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" output="screen">
    <param name="publish_frequency" type="double" value="50.0" />
  </node>
  
</launch>
